autoload/go/ui.vim

```diff
@@ -23,37 +28,60 @@ function! go#ui#OpenWindow(title, content)
     else
         exe 'resize ' . len(a:content)
     endif
```

Comment:
We should resize the window to the length of the content. I remember making it here to defaulting to `10`. But I think we can for now remove that limitation and make it in the future optional. So these lines:

```
    " Keep minimum height to 10, if there is more just increase it that it
    " occupies all results
    let buffer_height = 10
    if len(a:content) < buffer_height
        exe 'resize ' . buffer_height
    else
        exe 'resize ' . len(a:content)
    endif
```

should be changed to:

```
    " Resize window to content lenght
    exe 'resize ' . len(a:content)
```